sdk/storage/Azure.Storage.Queues/src/Sas/QueueSasBuilder.cs

```diff
@@ -110,7 +110,8 @@ public QueueSasBuilder()
         /// class.
         /// </summary>
         /// <param name="permissions">
-        /// The time at which the shared access signature becomes invalid.
+        /// The permissions associated with the shared access signature. 
```

Comment:
We need to remove the trailing whitespace so CI can pass.

Reply:
@seanmcc-msft Removed the trailing whitespace. Could you please have a look ?